agents/dnsagent.go

```diff
@@ -275,13 +276,13 @@ func (da *DNSAgent) processRequest(reqProcessor *config.RequestProcessor,
 	}
 	if reqProcessor.Flags.HasKey(utils.MetaLog) {
 		utils.Logger.Info(
-			fmt.Sprintf("<%s> LOG, Diameter reply: %s",
-				utils.DiameterAgent, agReq.Reply))
+			fmt.Sprintf("<%s> LOG, DNSAgent reply: %s",
+				utils.DNSAgent, agReq.Reply))
 	}
 	if reqType == utils.MetaDryRun {
 		utils.Logger.Info(
-			fmt.Sprintf("<%s> DRY_RUN, Diameter reply: %s",
-				utils.DiameterAgent, agReq.Reply))
+			fmt.Sprintf("<%s> DRY_RUN, DNSAgent reply: %s",
```

Comment:
please remove DNSAgent string from log string since it will look like repetition